content/en/docs/setup/additional-setup/external-controlplane/index.md

```diff
@@ -522,11 +482,15 @@ including gateways, if needed.
 
     {{< text bash >}}
     $ curl -s "http://${GATEWAY_URL}/hello"
-    Hello version: v1, instance: helloworld-v1-5b75657f75-ncpc5
+    Hello version: v1, instance: helloworld-v1-776f57d5f6-s7zfc
     {{< /text >}}
 
 ## Adding clusters to the mesh (optional) {#adding-clusters}
 
+{{< warning >}}
```

Comment:
Does it make sense to have this match the `experimental` boilerplate, and maybe update that with the `experimental` link?